Dockerfile

```diff
@@ -15,16 +15,54 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
+###############################################################################
+## 1. Base (stable)                                                          ##
+###############################################################################
+
 FROM python:2.7
 
+# nodejs repo
+# detects distribution and adds the right repo
+# See: https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager#debian-and-ubuntu-based-linux-distributions
+RUN curl -sL https://deb.nodesource.com/setup | bash -
+
+# install requirements from repos
 RUN apt-get update && \
-    apt-get -qy install ssl-cert poppler-utils git subversion --fix-missing
+    apt-get -qy install poppler-utils git subversion nodejs mysql-client --fix-missing
+
+# install python requirements
+RUN pip install --upgrade pip && \
+    pip install mock
+
+# install nodejs requirements
+RUN npm update && \
+    npm install --silent -g bower less clean-css uglify-js requirejs
+
+
+###############################################################################
+## 2. Code (changing)                                                        ##
+###############################################################################
+
+# select proper requierments level
+#ENV REQUIREMENTS lowest
+#ENV REQUIREMENTS release
+ENV REQUIREMENTS devel
```

Comment:
My concern is not about the production image. It's only about that in my opinion the dev image should be using the release requirements instead of dev requirements. 

Note that:
- Devel requirements != requirements for developers
- Release requirements != requirements for production sites

rather:
- Devel requirements == Latest *unreleased* version of the requirements
- Release requirements != Latest released version of package on pypi.